### PR TITLE
SDL-0192 Added nav button capability

### PR DESCRIPTION
**Pull request by vlrm10**
This PR is **ready** for review.

### Testing Plan
[Describe how your changes will be tested]

### Summary
[Summary of PR changes]

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: ffw/ButtonsRPC.js [hunk 1]
```diff
@@ -179,44 +179,72 @@ FFW.Buttons = FFW.RPCObserver.create(
           break;
         }
         case 'Buttons.SubscribeButton': {
-          try {
-              const params = request.params;
-              const appID = params.appID
-              const resultCode = this.subscribeButton(appID, params.buttonName);
-              console.log("Button " + params.buttonName + " " + resultCode + " resultCode");
+          const buttonName = request.params.buttonName;
+          const appID = request.params.appID;
```
Comment (YuriiYefimenko) [inline ffw/ButtonsRPC.js]: Maybe it's better to destructure `buttonName` & `appId`?

File: ffw/ButtonsRPC.js [hunk 2]
```diff
@@ -179,44 +179,72 @@ FFW.Buttons = FFW.RPCObserver.create(
...
+          if(buttonName.includes("NAV_")) {
+            this.navButtonSubscriptionToggle(appID, buttonName, true);
+            const resultCode = SDL.SDLModel.data.resultCode.SUCCESS;
+            console.log("Button " + buttonName + " " + resultCode + " resultCode");
```
Comment (YuriiYefimenko) [inline ffw/ButtonsRPC.js:186]: Did you forget to remove console.log?

File: ffw/ButtonsRPC.js (hunk 1 above)
Comment (vlrm10) [reply ffw/ButtonsRPC.js]: @YuriiYefimenko Agree. Fixed in aba78752fc7a0d8058e1de1c10052b5fc2687c17

File: ffw/ButtonsRPC.js (hunk 2 above)
Comment (vlrm10) [reply ffw/ButtonsRPC.js:186]: @YuriiYefimenko No.This is for testing purposes. It's OK

**Review by LiliiaShlikhtLuxoft — APPROVED** [review]

**Review by YuriiYefimenko — APPROVED** [review]

**Pull request merged**